Correct usage of ellipsis (#18170)
mdn/content: files/en-us/mdn/structures/page_types/glossary_page_template/index.md

> ---
> _Remember to remove this whole explanatory note before publishing_

> The **TermBeingDefined** is … (include concise definition of the term being defined).

Reviewer: ```suggestion
The **TermBeingDefined** is _(include a concise definition of the term)_.
```
